build/scripts/preprocess.js

```diff
@@ -11,13 +11,17 @@ const semverSort = require('semver-sort');
 function getDirectories(srcpath) {
   return fs.readdirSync(srcpath).filter(file => fs.lstatSync(path.join(srcpath, file)).isDirectory());
 }
+
+const ROOT_DIR = `${ __dirname }/../../`;
+
 // collect all docs projects
 const data = [];
-const projects = getDirectories(`${ __dirname }/docs`);
+const projects = getDirectories(`${ ROOT_DIR }/docs`);
+
 projects.forEach(project => {
 
-  const channels = ['latest', 'stable', 'beta', 'alpha'];
-  const unsorted = getDirectories(`${ __dirname }/docs/${ project }`);
+  const channels = ['master', 'stable', 'beta', 'alpha'];
```

Comment:
if we were going to add docs for stable/beta/alpha in the future then they would be bumped to the top of the sorting, it will be ignored if there is no folder